libsolidity/formal/SMTEncoder.cpp

```diff
@@ -391,11 +391,20 @@ void SMTEncoder::endVisit(TupleExpression const& _tuple)
 	createExpr(_tuple);
 
 	if (_tuple.isInlineArray())
-		m_errorReporter.warning(
-			2177_error,
-			_tuple.location(),
-			"Assertion checker does not yet implement inline arrays."
-		);
+	{
+		createExpr(_tuple);
```

Comment:
There is another `createExpr(_tuple)` in line 391. Why are we doing it again?

Reply:
Will remove this one, thanks